testkit/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/testkit/PersistentEntitySpec.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.lagom.scaladsl.testkit
+
+import akka.actor.ActorSystem
+import com.lightbend.lagom.scaladsl.persistence.PersistentEntity
+import com.lightbend.lagom.scaladsl.playjson.JsonSerializerRegistry
+import com.lightbend.lagom.scaladsl.testkit.PersistentEntityTestDriver.Issue
+
+object PersistentEntitySpec {
+
+  /**
+    * An exception to be thrown if the issues after running a test are non-empty.
+    * @param entityId the entity which caused the issue.
+    * @param issues the sequence of issues
+    */
+  final case class PersistentEntityHasIssuesException(entityId: String, issues: Seq[Issue]) extends Exception {
+    override def getMessage: String = {
+      s"Persistent entity $entityId had ${issues.size} issue(s)...\nFirst issue: ${issues.head}"
+    }
+  }
```

Comment:
This exception will have to move to the PE code itself.

I think we should better print all `issues` instead of the `issues.head`.

Reply:
@ignasi35 I'm revisiting this now. When you say that it will need to move to PE code, do you mean the code that an implementer will write? Will the library user need to know about the exception?

Reply:
I got confused by the name of the `object` and just realised this is already part of the `testkit` API.

The only missing bit is improving the message to all `issues` are (pretty?) printed.

Reply:
@ignasi35 Perfect! I can clean that up. I have a new branch that has slightly more elegant code. I'll try to push that later.